engine/src/test/java/io/zeebe/engine/util/JobClient.java

```diff
@@ -112,17 +82,17 @@ public JobClient withErrorMessage(String message) {
   }
 
   public JobClient expectRejectionOnComplete() {
-    completeSupplier = COMPLETE_REJECTION_SUPPLIER;
+    completeSupplier = REJECTION_SUPPLIER;
```

Comment:
They can also be summarized then with `expectRejection()` and you only have on field `expectation`, which is then overwritten and used in the update, fail or complete